Umbraco-Cloud/Databases/Local-Database/index.md

```diff
@@ -11,15 +10,12 @@ This article will cover how you can connect to the database of your project that
 
 ## Connecting to your local Umbraco installation
 
-When cloning down your project to work locally you might want to have a look in your database every now and then. You can connect to your local Umbraco database through, for example, Visual Studio.
+When cloning down your project to work locally you might want to have a look in your database every now and then.
 
-In Visual Studio this is done through the Server Explorer. Add a new connection to a SQL server database file. Umbraco's database file can be found in `App_Data`:
+With the release of Umbraco 10, LocalDB is no longer supported and instead in Umbraco 10 we use SQLite.
+When you clone down your Umbraco project and restores its content it will create a `Umbraco.sqlite.db` file.
```

Comment:
```suggestion
With the release of Umbraco 10, LocalDB is no longer supported, and instead, in Umbraco 10 we use SQLite.
When you clone down your Umbraco project and restore its content it will create a `Umbraco.sqlite.db` file.
```